### PR TITLE
Adding announcement of Proposed Final Draft

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: news/2017-06-26-bean-validation-2-0-proposed-final-draft-released.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+== What's new in Bean Validation 2.0?
+
+The focus of Bean Validation 2.0 is supporting and taking advantage of Java SE 8.
+
+For instance type annotations are used to validate the elements of generic containers: `List<@NotNull @Email String> emails`.
+There are new built-in constraints (e.g. `@NotBlank`, `@NotEmpty`, `@Email`, `@Positive` and `@Negative`) and all built-in constraints are repeatable annotations now.
+Bean Validation 2.0 also supports the new Java 8 date and time types (JSR 310), the property types defined by JavaFX (`StringProperty` etc.) as well as `java.util.Optional`.
+
+To learn more about all the new functionality in Bean Validation 2.0,
```
Comment (gsmet) [inline news/2017-06-26-bean-validation-2-0-proposed-final-draft-released.adoc]: I have a doubt on `functionality` here (see https://www.merriam-webster.com/dictionary/functionality). Maybe `features` would be better?

Other than that, nothing to add.

Comment (gunnarmorling) [reply news/2017-06-26-bean-validation-2-0-proposed-final-draft-released.adoc]: I'll change it to "features", although the second definition in your link fits very well:

>  the set of functions or capabilities associated with computer software

Don't you think so?

**Comment by gunnarmorling** [comment]
Reworkred and force-pushed. Thanks for reviewing, @gsmet!

File: news/2017-06-26-bean-validation-2-0-proposed-final-draft-released.adoc (hunk 1 above)
Comment (gsmet) [reply news/2017-06-26-bean-validation-2-0-proposed-final-draft-released.adoc]: Ah, yes. Looks like they use the plural form in the example though.

**Pull request closed without merging**